lib/bolt/config.rb

```diff
@@ -147,6 +147,14 @@ def validate
       unless %w[human json].include? self[:format]
         raise Bolt::CLIError, "Unsupported format: '#{self[:format]}'"
       end
+
+      self[:transports].each_value do |v|
+        timeout_value = v[:connect_timeout]
+        unless timeout_value.is_a?(Integer) || timeout_value.nil?
+          error_msg = "connect-timeout value must be an Integer received #{timeout_value}:#{timeout_value.class}"
```

Comment:
I'd like a separator between `Integer` and `received`, either comma or `but`.

Reply:
Fixed